pilot/pkg/xds/ecds_test.go

```diff
@@ -240,6 +240,19 @@ func TestECDSGenerate(t *testing.T) {
 			wantExtensions:   sets.Set{"default.default-plugin-with-sec": {}},
 			wantSecrets:      sets.Set{"default-docker-credential": {}},
 		},
+		{
```

Comment:
Can we add some comments for this test case?
It might be something like:
```
// All the credentials should be sent to istio-agent even if one of them is only updated,
// because `istio-agent` does not keep the credentials.
```